### ISSUE TITLE
Update immersive page icon buttons (back, close) for screen reader usage

### ISSUE THREAD

**Issue by marcellamaki** [learningequality/kolibri#9407]
## Observed behavior
As [reported in the support forum ](https://community.learningequality.org/t/nvda-screen-reader-is-not-able-the-kolibri-desktop-application/2417/3), the close and back button on (at least some, perhaps all) immersive pages in Kolibri do not have correct icon identification information.

## Expected behavior

Immsersive page icon buttons should be clearly labeled for screen reader users.

_n.b._: After investigation, it may be that this requires a KDS update, either in addition to, or instead of, a Kolibri implementation change.

## User-facing consequences
Screen reader users will have trouble navigating out of their current page.

## Context
Chrome, 0.15.1

**Comment by rtibbles** [comment]
Fixed in #9411
